### PR TITLE
Create gerg1

### PR DESCRIPTION
**Pull request by gerg1**

### CODE REVIEW

File: _pins/gerg1 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: gerg1
```
Comment (crichID) [inline _pins/gerg1]: @gerg1 you will need to add the `.json` extension to this file in order for your pin to render correctly.

Comment (crichID) [inline _pins/gerg1]: @gerg1 you will also need to add a line containing `---` to the beginning and end of this file.

File: _pins/gerg1 [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+latitude: 46.072735
+longitude:18.232266
```
Comment (crichID) [inline _pins/gerg1]: @gerg1 you will need to add a space between the : and the 18 for longitude.

**Comment by brianamarie** [comment]
@gerg1 I am going to close this pull request. Please re-open when you are ready to fix based on Cynthia's comments. 😄

**Pull request closed without merging**